analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

```diff
@@ -0,0 +1,386 @@
+
+transcriptomic_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+matched_samples_rna <- length(unique(transcriptomic_assays_matched$Kids_First_Participant_ID))
+```
+## How many samples have transcriptomic assays and matched longitudinal information?
+There are `r matched_samples_rna` patient cases with matched longitudinal samples for transcriptomic assays.
+
+# Paired genomic and transcriptomic assays
+We will look only into paired assays now.
+```{r}
+paired_assays <- tumor_descriptor_df %>%
+  filter(experimental_strategy_sum == "Both") %>%
+  group_by(Kids_First_Participant_ID) %>%
+  summarize(descriptors = paste(sort(tumor_descriptor), collapse = ", "),
+            experimental_strategy = paste(sort(experimental_strategy), collapse = ", "),
+            cancer_group = paste(sort(cancer_group), collapse = ", "))
+
+kids_pbta_samples_paired <- length(unique(paired_assays$Kids_First_Participant_ID))
+```
+## How many samples have paired genomic and transcriptomic assays?
+There are `r kids_pbta_samples_paired` patient cases with paired genomic and transcriptomic assays.
+
+## All time points
+```{r}
+paired_assays %>%
+  group_by(descriptors) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+paired_assays %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+paired_assays_matched <- paired_assays %>%
+  filter(descriptors %in% c("Diagnosis, Recurrence", "Diagnosis, Progressive", "Deceased, Diagnosis", 
+                            "Recurrence, Second Malignancy", "Progressive, Recurrence", "Diagnosis, Second Malignancy", 
+                            "Deceased, Progressive, Recurrence", "Diagnosis, Progressive, Recurrence", "Deceased, Recurrence", "Deceased, Diagnosis, Recurrence",
+                            "Diagnosis, Unavailable")) %>%
+  write_tsv(file.path(results_dir, "paired_assays_matched_time_points.tsv"))
+
+paired_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+matched_samples_paired <- length(unique(paired_assays_matched$Kids_First_Participant_ID))
+```
+## How many samples have paired genomic and transcriptomic assays and matched longitudinal information?
+There are `r matched_samples_paired` patient cases with matched longitudinal samples for genomic and transcriptomic assays.
+
+# Genomic assays and cancer_group
+Let us investigate the number of tumor samples with genomic assays per cancer_group.
+```{r}
+list <- genomic_assays %>%
+  count(cancer_group) %>%
+  arrange(desc(n))
```

Comment:
One thing to keep in mind is that value of `cancer_group` group varies based on number of time points, even when patients belong to the same cancer group. For example, an HGG patients with one sample will have `High-grade glioma`, but an HGG patient with two time points will be `High-grade glioma, High-grade glioma` so this isn't really getting a breakdown of # patients by cancer group. This could be resolved by collapsing the `cancer_group` column to unique values when creating `genomic_assays`, but I'm not sure if that's what you want to do.

Reply:
Then maybe use `harmonized` or `broad_histology` instead will be more appropriate in this case?
@jharenza